content/sensu-go/5.0/reference/rbac.md

```diff
@@ -88,134 +235,409 @@ username     |
-user associated with it full access to the system.
+_WARNING: Modification of the `system:agents` group can result in non-functional Sensu agents._
 
-#### Attributes
+### Assigning a user to a group
+
+Groups are created and managed within user definitions.
+You can use [sensuctl][2] to add users to groups.
+
+To add a user to a group:
+
+{{< highlight shell >}}
+sensuctl user add-group [USERNAME] [GROUP]
+{{< /highlight >}}
+
+To set the groups for a user:
+
+{{< highlight shell >}}
+sensuctl user set-groups USERNAME GROUP1[,GROUP2, ...[,GROUPN]]
+{{< /highlight >}}
+
+### Removing a user from a group
+
+You can use [sensuctl][2] to remove users from groups.
+
+To remove a user from a group:
+
+{{< highlight shell >}}
+sensuctl user remove-group [USERNAME] [GROUP]
+{{< /highlight >}}
+
+To remove a user from all groups:
+
+{{< highlight shell >}}
+sensuctl user remove-groups [USERNAME]
+{{< /highlight >}}
+
+## Roles and cluster roles
+
+A role is a set of permissions controlling access to Sensu resources.
+**Roles** specify permissions for resources within a namespace while **cluster roles**  can include permissions for [cluster-wide resources][18].
+You can use [roles bindings][23] to assign roles to user and groups.
+To avoid re-creating commonly used roles in each namespace, [create a cluster role][28] and use a [role binding][29] (not a cluster role binding) to restrict permissions within a specific namespace.
+
+To create and manage roles cluster-wide, [configure sensuctl][26] as the [default `admin` user][20] or create a [cluster role][21] with `roles` permissions.
+To create and manage roles within a namespace, [create a role][25] with `roles` permissions within that namespace.
+
+### Cluster roles
+
+Cluster roles can specify access permissions for [cluster-wide resources][18] like users and namespaces as well as [namespaced resources][17] like checks and handlers.
+Cluster roles use the same [specification][24] as roles and can be managed using the same sensuctl commands with `cluster-role` substituted for `role`.
+
+To create and manage cluster roles, [configure sensuctl][26] as the [default `admin` user][20] or [create a cluster role][25] with permissions for `clusterroles`.
+
+### Default roles
+
+Every [Sensu backend][1] includes:
+
+| Role name       | Type          | Description |
+| --------------- | ------------- | ----------- |
+| `cluster-admin` | `ClusterRole` | Full access to all [resource types][4] across namespaces, including access to [cluster-wide resource types][18].
+| `admin`         | `Role`        | Full access to all [resource types][4] within a namespace. |
+| `edit`          | `Role`        | Read and write access to most resources within a namespace with the exception of roles and role bindings.
+| `view`          | `Role`        | Read-only permission to most [resource types][4] within a namespace with the exception of roles and role bindings.  |
```

Comment:
The `admin`, `edit` & `view` roles are all `ClusterRole`. The distinction here is that these 3 cluster roles are intended to be used via a Role Binding, so they act as a kind of template for each namespace.